src/IO.go

```diff
@@ -56,3 +66,14 @@ func makeDir(name string) {
 		log.Error(err)
 	}
 }
+
+func makeRed(input string) string {
+	tokens := strings.Split(input, "\n")
+	for i, row := range tokens {
+		if row == "" {
```

Comment:
No need to.
Format of the ascii table has been changed to never altering the cells formatting in favor of format sensitive output. This way No special attention is needed for inline structures as the whole line is colored.

One question tho: Is it practical to search for \r as well? according to e.g. http://stackoverflow.com/questions/1761051/difference-between-n-and-r
\r is, if anything, used on old systems and electromechanical teletype-like "terminals"